src/inventory_ui.cpp

```diff
@@ -2119,6 +2152,7 @@ std::string inventory_selector::action_bound_to_key( char key ) const
 item_location inventory_pick_selector::execute()
 {
     shared_ptr_fast<ui_adaptor> ui = create_or_get_ui_adaptor();
+    ui_manager::redraw(); // Needed for highlight to display before any key is pressed.
```

Comment:
When you resize the window the highlight also disappears. I suggest removing this call and moving the "highlight" call to the `refresh_window` function and see if highlight is displayed correctly when first displaying the screen and when resizing the window.

Reply:
Since it is really worthy of another PR, it is probably best to leave this out. Will remove this line.